Connection/Handler.php

```diff
@@ -263,8 +264,11 @@ public function send($message, Socket\Node $node = null)
             $self = $this;
 
             return function () use (&$send, &$old, &$self) {
-                $out = call_user_func_array($send, func_get_args());
-                $self->getConnection()->_setStream($old);
+                try {
```

Comment:
Actually, we can apply the same behavior with the first `$this->_send(…)` too. I am making this change.